config/initializers/convert_time_to_eo.rb

```diff
@@ -0,0 +1,11 @@
+# temporary hack for when/if we can get patch accepted upscream
+# see note at https://github.com/ManageIQ/manageiq/pull/19153#issuecomment-521719495
```

Comment:
All joking aside, we probably just need to say that et-orbi doesn't support comparisons of `EoTime` in the scheduler with user provided `Time` objects.  Other than the spec comparisons, what else does this fix?

Reply:
It's primarily just spec comparisons in the runner spec and in the scheduler spec.